### PR TITLE
ScrollView: fix native scrolling bugs

### PR DESCRIPTION
**Pull request by nekronos**
This PR contains:
- [x] Changelog
- [ ] Documentation
- [ ] Tests

### CODE REVIEW

File: Source/Fuse.Controls.Native/Interfaces.uno [hunk 1]
```diff
@@ -123,6 +123,7 @@ namespace Fuse.Controls.Native
 	public interface IScrollViewHost
 	{
 		float PixelsPerPoint { get; }
+		float2 ContentSize { get; }
```
Comment (mortoray) [inline Source/Fuse.Controls.Native/Interfaces.uno]: This needs a CHANGLEOG entry... though I suspect this interface should actually be internal.

Comment (nekronos) [reply Source/Fuse.Controls.Native/Interfaces.uno]: I'll add that. But yeah, everything in that file should have been `internal`

Comment (mortoray) [reply Source/Fuse.Controls.Native/Interfaces.uno]: Or chnage them to internal and note that in the CHANGELOG.

Comment (kusma) [reply Source/Fuse.Controls.Native/Interfaces.uno]: Yeah, I think IScrollViewHost should be made internal.

Comment (nekronos) [reply Source/Fuse.Controls.Native/Interfaces.uno]: `IScrollViewHost` is used in public `UXConstructor`'s. Cannot easily be made internal

Comment (nekronos) [reply Source/Fuse.Controls.Native/Interfaces.uno]: I'll do a pass on `Fuse.Control.Native` and make whats need to be internal internal and make a separate PR

**Comment by nekronos** [comment]
Fixes https://github.com/fusetools/fuselibs-public/issues/917

**Pull request merged**